data/Course Student List/CST232 - OPERATING SYSTEMS.txt

```diff
@@ -0,0 +1,2 @@
+Student 1 Name	123456
```

Comment:
Why got students in this file? Shouldn't it be added when students register or is it for testing purpose, if not, it's better to remove it until it is used, correct me if I'm wrong

Reply:
If you check "data\Student Registered Course\123456.txt" this student has already registered for 5 courses:
- CST232 - OPERATING SYSTEMS
- MAT111 - LINEAR ALGEBRA
- CST131 - COMPUTER ORGANISATIONS
- CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP
- LSP300 - ACADEMIC ENGLISH

So in "data/Course Student List/CST232 - OPERATING SYSTEMS.txt" should have this student name and matric number becuase he registered for this subject already.

Reply:
If it is a testing file can delete ya